### PR TITLE
Theme + Visual refresh ✨

### PR DESCRIPTION
**Pull request by peduarte**
This PR Updates
- Color tokens
- Switch visual
- Select visual
- Input visual
- Textarea visual
- Slider visual
- GhostButton visual
- Button visual
- ToggleButton visual
- Link visual

### CODE REVIEW

**Comment by vladmoroz** [comment]
Nice!

Do we want to merge it when we also add the greens, reds, and yellows? I have a working version of all colors already, expecting to finalize it with Colm tomorrow or so.

**Comment by peduarte** [comment]
@vladmoroz Let's wait until we update the other colors then. Let me know 👍

File: packages/radix/src/components/Badge.tsx [hunk 1]
```diff
@@ -29,11 +29,12 @@ export const BaseBadge = React.forwardRef<HTMLSpanElement, BadgeProps>((props, r
           height: 3, // TODO: Remove `inline-flex` from Primitives and then we can make this `inline-block` and remove height
           minWidth: 5,
           paddingY: 0,
-          paddingX: 1,
+          paddingX: '1.25ch',
```
Comment (peduarte) [inline packages/radix/src/components/Badge.tsx]: Can we not rely on the scale for this? 🙄

Comment (vladmoroz) [reply packages/radix/src/components/Badge.tsx]: Hey, neither scale value worked well. The space is so tight here that every pixel makes a difference.

It’s sort of typography related: the monospace font has pretty large spaces between words, so badge side paddings need to be a bit larger than the space character to look balanced. Too small — say, 5 px like we had, and badges with more than one word fall apart visually. More — and it looks elongated, sort of unbalanced compared to its small font.

Hence `ch` the unit. It should age well if we add another size of the badge or change its font size. Do you mind if we keep it this way?

Comment (peduarte) [reply packages/radix/src/components/Badge.tsx]: Fair enough, I'm ok keeping this, we already use `em` in some places anyway. Also this is the perfect use case for `ch` so let's go for it. Thanks!

**Pull request merged**